src/script/components/groupVideoGrid.ts

```diff
@@ -31,10 +31,13 @@ interface GroupVideoGripParams {
 
 class GroupVideoGrid {
   private readonly grid: ko.PureComputed<Grid>;
+  private readonly videoParticipants: ko.PureComputed<Participant[]>;
   private readonly minimized: boolean;
   public readonly muted: ko.Observable<boolean>;
   public readonly selfParticipant: Participant;
   public readonly dispose: () => void;
+  private readonly maximizedParticipant: ko.Observable<Participant>;
```

Comment:
```suggestion
  private readonly maximizedParticipant: ko.Observable<Participant | null>;
```

Further in your code you have this condition `this.maximizedParticipant() !== null` so this should be reflected in the type.